examples/management/managementgroupsubscriptionassociation.yaml

```
@@ -0,0 +1,43 @@
+apiVersion: management.azure.upbound.io/v1beta1
+kind: ManagementGroupSubscriptionAssociation
+metadata:
+  annotations:
+    meta.upbound.io/example-id: management/v1beta1/managementgroupsubscriptionassociation
```

```suggestion
    meta.upbound.io/example-id: management/v1beta1/managementgroupsubscriptionassociation
    upjet.upbound.io/manual-intervention: "This resource is skipping because depends on Subscription resource"
```
nit pick: We should add manual intervention to all dependent resources.